add ai concepts section (#8087)
aws-amplify/docs: src/pages/[platform]/ai/concepts/models/index.mdx

### AI21 Labs
* [Jamba 1.5 Large](https://aws.amazon.com/blogs/aws/jamba-1-5-family-of-models-by-ai21-labs-is-now-available-in-amazon-bedrock/)
* [Jamba 1.5 Mini](https://aws.amazon.com/blogs/aws/jamba-1-5-family-of-models-by-ai21-labs-is-now-available-in-amazon-bedrock/)

### Anthropic
* Claude 3 Haiku
* Claude 3 Sonnet
* Claude 3 Opus
* Claude 3.5 Sonnet
https://docs.anthropic.com/en/docs/about-claude/models

### Cohere
* Command R
* Command R+

### Meta
* Llama 3.1

### Mistral
* Large
* Large 2

The Amplify AI Kit makes use of ["tools"](/[platform]/ai/concepts/tools) for both generation and conversation routes. [The models it supports must support tool use in the Converse API](https://docs.aws.amazon.com/bedrock/latest/userguide/conversation-inference-supported-models-features.html). 

Using the Converse API makes it easy to swap different models without having to drastically change how you interact with them.

## Choosing a model

Each model and model provider has their own strengths and weaknesses. We encourage you to try different models for different use-cases to find the right fit. Things to consider when choosing a model:

### Context window

> Each model has its own context window size. The context window is how much information you can send to the model. Because FMs are stateless, but can contain message history, the context window can continue to grow as you "chat" with a model. The context window for models is defined by the number of tokens it can receive

Reviewer: ```suggestion
Each model has its own context window size. The context window is how much information you can send to the model. FMs are stateless, but conversation routes manage message history, so the context window can continue to grow as you "chat" with a model. The context window for models is defined by the number of tokens it can receive.
```
